seatunnel-connectors-v2/README.md

```diff
@@ -2,10 +2,11 @@
 This article introduces the new interface and the new code structure on account of the newly designed API for Connectors in Apache SeaTunnel. This helps developers with quick overview regarding API, translation layer improvement, and development of new Connector.
 
 ## **Code Structure**
-In order to separate from the old code, we have defined new modules for execution flow. This facilitates parallel development at the current stage, and reduces the difficulty of merging. All the relevant code at this stage is kept on the ``api-draft`` branch.
+In order to separate from the old code, we have defined new modules for execution flow. This facilitates parallel development at the current stage, and reduces the difficulty of merging.
 
 ### **Example**
 We have prepared a new version of the locally executable example program in ``seatunnel-examples``, which can be directly called using ``seatunnel-flink-connector-v2-example`` or ``seatunnel-spark-connector-v2-example`` in ``SeaTunnelApiExample``. This is also the debugging method that is often used in the local development of Connector. The corresponding configuration files are saved in the same module ``resources/examples`` folder as before.
+For now, the test needs to rely on the Connector,if your test must use a third-party jar and the Scope is  Provided,you need to check "Include dependencies with 'Provided Scope" and choose current Project in "Run"->"Edit configurations..."->"Use classpath of module" . 
```

Comment:
Currently, you need to add the dependencies of the corresponding connectors to the pom file for testing under the `seatunnel-examples` module, if there are some three-party dependencies, please add them to the module classpath by yourself, if the scope of some dependencies of the connectors is `provided`, you also need to set `Include dependencies with Provided Scope` in Intellij IDEA.